src/main/kotlin/io/emeraldpay/dshackle/upstream/ethereum/EthereumEgressSubscription.kt

```diff
@@ -27,23 +28,22 @@ open class EthereumEgressSubscription(
         const val METHOD_PENDING_TXES = "newPendingTransactions"
     }
 
-    private val availableTopics = listOf(
-        METHOD_NEW_HEADS,
-        METHOD_LOGS,
-        METHOD_SYNCING,
```

Comment:
Maybe it useless, but we lost it

Reply:
Then we can remove all code about syncing 😀

Reply:
ok